x-pack/filebeat/docs/inputs/input-awscloudwatch.asciidoc

```diff
@@ -8,7 +8,7 @@
 === awscloudwatch input
```

Comment:
You should also change the topic title.

Reply:
Done! I also changed the s3 input to S3 input. Thank you!